editor/editor_node.cpp

```diff
@@ -3661,8 +3661,6 @@ Ref<Texture2D> EditorNode::get_class_icon(const String &p_class, const String &p
 		if (icon.is_null()) {
 			icon = gui_base->get_theme_icon(ScriptServer::get_global_class_base(name), "EditorIcons");
 		}
-
-		return icon;
```

Comment:
Since this is removed, what is the whole point of this if block?

Reply:
Good point, lines 3661-3663 are useless code now.